src/blindpsbt.cpp

```diff
@@ -476,6 +476,9 @@ BlindingStatus BlindPSBT(PartiallySignedTransaction& psbt, std::map<uint32_t, st
                         }
                     }
                 }
+                else
+                    input_asset_blinders.emplace_back();
```

Comment:
In the case of unblinded issuances, we need to add a null value here for a valid surjection proof to exist.

Reply:
can we add the missing `{}` here?